.github/scripts/test_tekton_tasks.sh

```diff
@@ -103,13 +103,12 @@ do
 
     PIPELINERUN=$(tkn p start $TEST_NAME -w name=tests-workspace,volumeClaimTemplateFile=$WORKSPACE_TEMPLATE -o json | jq -r '.metadata.name')
     echo "  Started pipelinerun $PIPELINERUN"
-    sleep 1  # allow a second for the pr object to appear (including a status condition)
```

Comment:
I am pretty sure @mmalina did this for a reason. See https://github.com/redhat-appstudio/release-service-bundles/pull/177/files

Reply:
No idea how this ended there.